### ISSUE TITLE
Avatar links broken

### ISSUE THREAD

**Issue by matthewbcool** [mozilla/hubs#5732]
**Description**
Avatar links appear broken when you navigate to them on hubs.mozilla.com

```
Uncaught ReferenceError: NAF is not defined
    at 34105 (netcode.js:90:1)
    at l (bootstrap:19:32)
    at startup:4:95
    at l.O (chunk loaded:25:12)
    at startup:5:43
    at startup:5:45
```

**To Reproduce**
1. Go to [Space Person Avatar](https://hubs.mozilla.com/avatars/2BtycVC) or any avatar link.
2. See error

**Expected behavior**
Avatar links should not give us a blank screen.

**Hardware** 
 - Device: Macbook
 - Browser: firefox, chrome

**Comment by Pattersonbl2** [comment]
I just did a review on this issue.  It looks to be a configuration issue with HMC running in GCP.   
I see HMC loading the glb file however, hub-cloud doesn't.  Hubs-cloud loads a gltf plain text avatar name and 4 components that make up the avatar.  
<img width="494" alt="Screen Shot 2022-09-30 at 11 49 01 AM" src="https://user-images.githubusercontent.com/28871018/193361001-b51d56ef-bd2a-4d2f-9f51-958cfa4d0aa5.png">
<img width="515" alt="Screen Shot 2022-09-30 at 11 40 05 AM" src="https://user-images.githubusercontent.com/28871018/193361017-6d59e774-7753-47af-a796-a556ec9b323b.png">
The first picture is HMC and the second Is hubs-cloud.

**Comment by sarhugo** [comment]
I'm starting to check it but did you see this warning? Maybe there's a missing lib

![image](https://user-images.githubusercontent.com/974762/193711763-819287c3-bdff-48ce-9b28-411911901b62.png)

**Comment by sarhugo** [comment]
I see that this commit introduced the warning and seems to have fixed the issue https://github.com/mozilla/hubs/commit/804b9c36bdd782e4f0baa40000be39187fdde595

**Comment by matthewbcool** [comment]
patched with #5736
